js/src/dashboard/all-programs-table-card/index.js

```diff
@@ -113,15 +113,18 @@ const AllProgramsTableCard = ( props ) => {
 							el.id === FREE_LISTINGS_PROGRAM_ID ? (
 								<FreeListingsDisabledToggle />
 							) : (
-								<ProgramToggle program={ el } />
+								<ProgramToggle key={ el.id } program={ el } />
 							),
 					},
 					{
 						display: (
 							<div className="program-actions">
 								<EditProgramButton programId={ el.id } />
 								{ el.id !== FREE_LISTINGS_PROGRAM_ID && (
-									<RemoveProgramButton programId={ el.id } />
+									<RemoveProgramButton
+										key={ el.id }
+										programId={ el.id }
+									/>
 								) }
 							</div>
```

Comment:
💅  Instead of putting `key={ el.id }` at the `RemoveProgramButton`, I would put it at the top-most `div` wrapping everything inside.

Reply:
Moved by https://github.com/woocommerce/google-listings-and-ads/pull/771/commits/77848d9714f4ecff3be1150be85ed144a6c5a5ea and added comments for this fix.